autogalaxy/plot/mat_wrap/get_visuals.py

```diff
@@ -0,0 +1,451 @@
+from typing import List, Union
+
+import autoarray as aa
+
+from autoarray.plot.mat_wrap import get_visuals as gv
+
+from autogalaxy.plot.mat_wrap.include import Include1D
+from autogalaxy.plot.mat_wrap.include import Include2D
+from autogalaxy.plot.mat_wrap.visuals import Visuals1D
+from autogalaxy.plot.mat_wrap.visuals import Visuals2D
+
+from autogalaxy.util import error_util
+
+from autoarray.structures.grids.two_d.grid_2d import Grid2D
+from autoarray.structures.grids.two_d.grid_2d_irregular import Grid2DIrregular
+
+from autogalaxy.imaging.fit_imaging import FitImaging
+from autogalaxy.profiles.light_profiles.light_profiles import LightProfile
+from autogalaxy.profiles.mass_profiles.mass_profiles import MassProfile
+from autogalaxy.galaxy.galaxy import Galaxy
+from autogalaxy.plane.plane import Plane
+
+
+class GetVisuals1D(gv.GetVisuals1D):
+    def __init__(self, include: Include1D, visuals: Visuals1D):

```

Comment:
Pointless constructor. Put the docs under the class signature

Reply:
Its not pointless because its typed with different objects (e.g. autogalaxy.Include1D, not autoarray.Include1D)?

Reply:
Yeah I guess that does something in terms of linting